website/api/controllers/webhooks/receive-from-customer-fleet-instance.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
 
     webhookSecret: {
       type: 'string',
-      required: true,
+      required: false,
```

Comment:
Why does this PR make the webhook secret non-required?  The webhook secret should be provided.

For testing, contributors can use a fake webhookSecret request param value, and configure the same fake secret in the config.

cc @eashaw

Reply:
@mikermcneil yeah, this was from local testing and it slipped into the PR. I haven't worked with the website before, so I was trying lots of different things while testing to get a feel for how it worked and missed reverting this change. Fixed!